### ISSUE TITLE
SwiftLint rule violations

### ISSUE THREAD

**Issue by bdube** [feditext/feditext#2]
- [x] Superfluous disable commands
- [x] Discouraged direct init
- [ ] Will never execute

**Comment by VyrCossont** [comment]
FWIW the majority of existing "will never execute" warnings are Combine-related and are stub closures necessary only to give the publisher the correct type. We can turn warnings off for those lines.

We should also turn off warnings for TODOs, since I use those extensively to keep track of future work at the source level. If it's possible to warn on FIXME but not TODO, that would be best, since Xcode shows them differently and I use FIXME for stuff that should not be committed.

**Comment by sbeitzel** [comment]
Sadly, todo and fixme are not differentiated by SwiftLint. https://realm.github.io/SwiftLint/todo.html

**Comment by VyrCossont** [comment]
@sbeitzel huh? The config key that controls that behavior is on the page you linked.

**Comment by sbeitzel** [comment]
No, right, but that key controls *both* todo and fixme, so either we'd warn on both or on neither. That's all I meant.

**Comment by sbeitzel** [comment]
Oh, jeez. I'm sorry, I'm not used to deciphering YAML. Yes, I see how to only warn on FIXME and not on TODO.
